Refit/RequestBuilderImplementation.cs

```diff
@@ -446,6 +463,13 @@ public RestMethodInfo(Type targetInterface, MethodInfo methodInfo, RefitSettings
             Name = methodInfo.Name;
             MethodInfo = methodInfo;
 
+            var uma = methodInfo.GetCustomAttributes(true)
+                .OfType<UniqueNameAttribute>()
+                .FirstOrDefault();
+
+            if (uma != null)
+                Name = uma.MethodName;
```

Comment:
`UniqueNameAttribute` should not be compulsory for supporting overloads
name should default to method mangling #85 or [#82 Comment](https://github.com/paulcbetts/refit/issues/82#issuecomment-66140879)